client/components/Slide/Components/CheckResponse/Editor.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, TextArea, Label, Input } from 'semantic-ui-react';
+import { type } from './type';
+import './CheckResponse.css';
+
+class CheckResponseEditor extends React.Component {
+    constructor(props) {
+        super(props);
+        const {
+            prompt = 'Text Prompt (displayed before input field as label)',
+            responseId = 'response-1',
+            checked,
+            unchecked
+        } = props.value;
+        this.state = {
+            prompt,
+            checked,
+            unchecked,
+            responseId
+        };
+
+        this.onTextAreaChange = this.onTextAreaChange.bind(this);
+    }
+
+    render() {
+        const { prompt, checked, unchecked, responseId } = this.state;
+        const { onTextAreaChange } = this;
+        return (
+            <React.Fragment>
+                <Form>
+                    <Label as="label" className="CheckResponse-label">
+                        <p>Response ID:</p>
+                        <Input
+                            name="responseId"
+                            value={responseId}
+                            onChange={onTextAreaChange}
+                            className="CheckResponse-input"
+                        />
+                    </Label>
+                    <Label as="label" className="CheckResponse-label">
+                        <p>Label:</p>
+                        <TextArea
+                            name="prompt"
+                            value={prompt}
+                            onChange={onTextAreaChange}
+                            className="CheckResponse-input"
+                        />
+                    </Label>
+                    <Label as="label" className="CheckResponse-label">
+                        <p>Unchecked (default) value:</p>
```

Comment:
Do we want to give TM authors to specify a different value for whether the box is checked or unchecked? I think it might be a little confusing to need to specify both, especially when the traditional HTML checkbox is only ever "checked" or not.

Reply:
So - 99% of the time I've used a traditional checkbox I've used an `<input name="check" type="hidden" value="unchecked"><input name="check" type="checkbox" value="checked">` to get around that --- Almost all "form" usages have a checked and unchecked value for checkboxes IMO

Reply:
Okay, if you think that will be helpful for data collection. Having checked and unchecked values is new to me 👍

Reply:
I think it's way more helpful than having your form response from a checkbox be `""`

Reply:
@gnarf I'm not seeing the checked and unchecked values being persisted or saved at all. Is that for the next iteration?

Reply:
nope - copy paste bug - nice catch - fixing

Reply:
741bfea23f59e6184595d43f971bd8cda003f755